src/cider/nrepl/middleware/test.clj

```diff
@@ -353,6 +355,11 @@
         (t/send transport (response-for msg :status :no-error)))
       (t/send transport (response-for msg :status :done)))))
 
+(defn handle-diff-backend [handler {:keys [diff-backend]
```

Comment:
I don't think that this is needed. Probably we can just have clients send via the test requests what diff backend to to use or something like this. A separate middleware just to add some key to a message seems a bit excessive to me.